manager/src/main/java/io/gingersnapproject/configuration/LazyRule.java

```diff
@@ -4,11 +4,11 @@
 
 import io.smallrye.config.WithDefault;
 
-public interface Rule {
+public interface LazyRule {
```

Comment:
The `connector`, `queryEnabled` and `expandEntity` methods can be removed as these are not relevant to LazyCacheRules.

Reply:
These methods are referred in some places:
- IndexingHandler
- DatabaseHandler
- Caches

Do they only work with EagerRules? Or we need to dig into case by case?